### ISSUE TITLE
Invalid cookie header: Invalid 'expires' attribute messages flood the build output

### ISSUE THREAD

**Issue by maxbog** [gradle/gradle#1596]
When downloading jars from custom repository running on AWS, the following messages are displayed:

Invalid cookie header: "Set-Cookie: AWSALB=gQoLH+yfJ5Q1FtG42X6LxcXpElV3tGjbRRtKVkdgpzbUi6VvvAUUTFxZddt28f7EIwwNEv0SJRl+itg6OKSQcRnc9tOc2TWlrye/cxrXoci0veb/c3J+RIYZEGyQ; Expires=Thu, 23 Mar 2017 15:03:12 GMT; Path=/". Invalid 'expires' attribute: Thu, 23 Mar 2017 15:03:12 GMT

One message per jar appears in the output, so basically the screen is flooded with them

### Expected Behavior
Such messages should not be visible in the output

### Current Behavior
A multitude of:
Invalid cookie header: "Set-Cookie: AWSALB=[COOKIE GOES HERE]; Expires=Thu, 23 Mar 2017 15:03:12 GMT; Path=/". Invalid 'expires' attribute: Thu, 23 Mar 2017 15:03:12 GMT

### Context
We have a custom maven repository running on AWS with all packages downloaded from there.

This may be related: https://issues.apache.org/jira/browse/HTTPCLIENT-1077
Especially if gradle uses apache HttpClient

**Comment by arakelian** [comment]
This happens for me against Oracle Maven Repository

**Comment by KidDaedalus** [comment]
This happens for me when attempting to push to a Gradle Enterprise based remote build cache

**Comment by wolfs** [comment]
The PR has been merged and the fix will be included in 4.1 RC1.

**Comment by ajeetmahajan** [comment]
Hello,

I am getting same error as below:
Invalid cookie header: "Set-Cookie: AWSALB=gQoLH+yfJ5Q1FtG42X6LxcXpElV3tGjbRRtKVkdgpzbUi6VvvAUUTFxZddt28f7EIwwNEv0SJRl+itg6OKSQcRnc9tOc2TWlrye/cxrXoci0veb/c3J+RIYZEGyQ; Expires=Thu, 23 Mar 2017 15:03:12 GMT; Path=/". Invalid 'expires' attribute: Thu, 23 Mar 2017 15:03:12 GMT

Could somebody help me to fix this? How to upgrade Apache HTTP Client?

Thanks,
Ajeet Mahajan

**Comment by pranaykumargoud** [comment]
Same here. Anyone please help

**Comment by omnilight** [comment]
Hi there.

The same problem for us. Using 
```
Gradle 5.6
```

got
```
Invalid cookie header: "Set-Cookie: AWSALB=W0GuZZn5L/EX9cbprGFG1/O4sIQoxl1rSkGvOaFtxFnPgHUiOj1IKHY3sv6jUk4FDljODKcvQqZ80F2m+I8tftxFh4EiRpi2UKZ7D2liGiT93ddW6vlKeKAGAzs3; Expires=Wed, 02 Oct 2019 08:40:53 GMT; Path=/". Invalid 'expires' attribute: Wed, 02 Oct 2019 08:40:53 GMT
build	25-Sep-2019 11:40:54	Versions fetched successfully.
```

**Comment by lisushka** [comment]
We've also started seeing this issue recently.  We had a successful build using Gradle 3.4.1 on the 20th of September.  When we tried to rebuild the same application on the 24th, we saw the following error:

```
Invalid cookie header: "Set-Cookie: <cookie>; Expires=Tue, 01 Oct 2019 06:03:13 GMT; Path=/". Invalid 'expires' attribute: Tue, 01 Oct 2019 06:03:13 GMT
```

This error is causing our CI builds to fail.  Does anyone have any idea how we might fix it?  (I'm not sure that upgrading the Gradle version will fix it, as we had identical code building successfully a week ago.)
